src/ArchiveAdmin.php

```diff
@@ -324,6 +331,13 @@ public function getManagedModelTabs()
         $forms = ArrayList::create();
         $mainModels = $this->getVersionedModels('main', true);
 
+        // Display order should be Pages > Blocks > Files > Other
+        // Pages, Blocks, Files are treated specially and have extensions defined in _config/archive-admin.yml
```

Comment:
Where do user defined tabs come into this? (Aside from "other")

Reply:
I'm not sure, I presume at either before or after the 3 special tabs.  I'm assuming that they add tabs via an extension and yml similar to https://github.com/silverstripe/silverstripe-versioned-admin/blob/1/_config/archive-admin.yml.  Probably after these 3 tabs though before 'Other'?  I would expect the uasort() function to not impact the user defined tabs.